google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleStage.java

```diff
@@ -120,9 +119,20 @@ public void stage(@NotNull Path stagingDirectory) {
       if (isCustomRuntime) {
         FileUtils.copyDirectory(Paths.get(dockerDirectory).toFile(), stagingDirectory.toFile());
       }
-    } catch (IOException | InvalidPathException | MalformedYamlFileException ex) {
-      loggingHandler.print(ex.getMessage() + "\n");
-      throw new RuntimeException(ex);
+    } catch (Exception e) {
```

Comment:
Oh interesting, I've never heard of that. In fact, this line from the Java Practices guide calls this out and says it's ok for a few certain applications: http://go/java-practices/naming#single-letter-variables-e-i-etc